Cubical/Relation/Binary/Preorder.agda

```diff
@@ -139,3 +140,51 @@ module PreorderReasoning (P' : Preorder ℓ ℓ') where
 
  infixr 0 _≤⟨_⟩_
  infix  1 _◾
+
+record isOrderEquiv (P : Preorder ℓ ℓ') (x y : ⟨ P ⟩) : Type ℓ' where
```

Comment:
Shouldn't this be called `OrderEquivalent`?

Reply:
Changed